pkg/crcontrollers/reservation/reservationcontroller.go

```diff
@@ -19,30 +23,55 @@ import (
 
 const (
 	contextTimeoutSeconds = 60
+
+	// reservation parameters
+	fastTimeoutEnv = "RESERVATION_FAST_TIMEOUT"
+	slowTimeoutEnv = "RESERVATION_SLOW_TIMEOUT"
+	attemptsNumEnv = "RESERVATION_ATTEMPTS_NUM"
+
+	defaulFastTimeout = 1500 * time.Millisecond
+	defaulSlowTimeout = 12 * time.Second
+	defaulAttemptsNum = 30
 )
 
 // Controller to reconcile aviliablecapacityreservation custom resource
 type Controller struct {
 	client                 *k8s.KubeClient
 	log                    *logrus.Entry
 	capacityManagerBuilder capacityplanner.CapacityManagerBuilder
+	fastTimeout            time.Duration
+	slowTimeout            time.Duration
+	maxAttempts            uint64
```

Comment:
Why `uint64` ? Use `int`

Reply:
@rkiselenko `maxAttempts` cannot be negative

Reply:
This validation should be on the `CRD` level, so negative number wont pass to controller. https://book.kubebuilder.io/reference/markers/crd-validation.html

Reply:
We have this check in the CRD. I wish to highlight here that `maxAttempts` is a natural number. Is it bad?